### PR TITLE
MIP-199 transform base schema implementation with test

### PR DESCRIPTION
**Pull request by SagarKarwande**
Add base implementation schema for transform action along with test.

### CODE REVIEW

File: apps/acqdat_core/lib/acqdat_core/streams/schema/transform.ex [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  def update_changeset(action, params) do
+    action
+    |> changeset(params)
+  end
+
+  # TODO validate the javascript code syntax.
+  defp changeset(action, params) do
+    action
+    |> cast(params, @all_params)
+    |> validate_required(@all_params)
```
Comment (SagarKarwande) [inline apps/acqdat_core/lib/acqdat_core/streams/schema/transform.ex:41]: Remove validate required

File: apps/acqdat_core/test/streams/schema/transform_test.ex [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      {:ok, action} = Repo.insert(Action.create_changeset(%Action{}, params))
+      [action: action]
+    end
+
+    test "valid changeset", %{action: action} do
+      params = %{
+        type: :transform,
+        name: "transform",
+        config: %{
+          destination: "return 1+2;"
```
Comment (SagarKarwande) [inline apps/acqdat_core/test/streams/schema/transform_test.ex:64]: Remove destination

**Comment by oyeb** [comment]
Rebasing on `stream/main`

**Review by oyeb — APPROVED** [review]
lgtm :+1:

**Pull request merged**